lib/utils/index.js

```diff
@@ -159,25 +157,25 @@ function textStyle(layer) {
       keywords.push(attr_value);
     }
   });
-  let letter_spacing = layer.letter_spacing;
-  if (letter_spacing) {
-    keywords.push(`letter_spacing_${letter_spacing}`);
-  }
-  let line_spacing = layer.line_spacing;
-  if (line_spacing) {
-    keywords.push(`line_spacing_${line_spacing}`);
-  }
-  if (font_size || font_family || !isEmpty(keywords)) {
-    if (!font_family) {
-      throw "Must supply font_family for text in overlay/underlay";
+
+  Object.keys(layer).forEach((attr) => {
+    if (attr === "letter_spacing" || attr === "line_spacing") {
+      keywords.push(`${attr}_${layer[attr]}`);
+    }
+    if (attr === "font_antialiasing" || attr === "font_hinting") {
+      keywords.push(`${attr.split("_").pop()}_${layer[attr]}`);
     }
-    if (!font_size) {
-      throw "Must supply font_size for text in overlay/underlay";
+    if (attr || !isEmpty(keywords)) {
```

Comment:
@strausr , I don't understand how this works

Reply:
changed logic in new commit